client/v2/example_test.go

```diff
@@ -10,7 +12,7 @@ import (
 )
 
 // Create a new client
-func ExampleClient() {
+func ExampleClient_HTTP() {
```

Comment:
Also, since you're removing the plain `ExampleClient()`, put a new simple example like this, also for the godocs

```
// Create a new client
func ExampleClient() {
	// Make client
	c, err := client.NewHTTPClient(client.HTTPConfig{
	    Addr: "http://localhost:8086",
	})
	if err != nil {
	    fmt.Println("Error creating InfluxDB Client: ", err.Error())
	}
	defer c.Close()
}
```